lib/guardian.rb

```diff
@@ -380,7 +380,7 @@ def can_invite_via_email?(object)
   end
 
   def can_bulk_invite_to_forum?(user)
-    user.admin?
+    authenticated? && user.admin?
```

Comment:
Hmm why do we need `authenticated?` here?

Reply:
Because it checks for presence of `user`, and `user.admin?` raises error if `user` is not present.

Reply:
Actually, we don't need to check for presence of user anymore because I updated `create_invite_link` method to check for `requires_login`. Will update accordingly.